community/kernel/src/main/java/org/neo4j/kernel/impl/api/OperationsFacade.java

```diff
@@ -1103,6 +1103,12 @@ public void uniqueIndexDrop( IndexDescriptor descriptor ) throws DropIndexFailur
         schemaWrite().uniqueIndexDrop( statement, descriptor );
     }
 
+    @Override
+    public RawIterator<Object[], ProcedureException> procedureCallSchema( ProcedureName name, Object[] input ) throws ProcedureException
+    {
+        return callProcedure( name, input, AccessMode.Static.FULL );
```

Comment:
Maybe now that we have procedureCallSchema we can lower the access mode of procedureCallWrite from FULL to WRITE (see the TODO in this file)